README.md

```diff
@@ -156,10 +156,7 @@ module.exports = class Item extends Model {
       options: {
         classMethods: {
           associate: (models) => {
-            models.Item.belongsToMany(models.User, {
```

Comment:
I don't wanna impose a way to do things about models, I removed the belongsToMany for avoiding confusion but I don't think was the best move.
Maybe we can expand owner configuration guidelines and specify that the requirement is having a relation with the user called owners. And pointing out that we can provide an easy-to-use n:m configuration or leave the implementation to the users
Can be a valid solution for you?